advising/urls.py

```diff
@@ -25,8 +25,21 @@
     url(r'^advising/jhu_signup/*$',
         FeatureFlowView.as_view(feature_name='JHU_SIGNUP', is_advising=True)),
     url(r'^advising/sis_post/$', advising.views.StudentSISView.as_view()),
+
+    # Get the semesters for the requesting user
     url(r'^advising/sis_semesters/$',
         advising.views.StudentSISView.as_view()),
-    url(r'^advising/sis_courses/(?P<sem_name>.+?)/(?P<year>[0-9]{4})/$',
+    # Get the semesters that the student with this JHED has imported from SIS
+    url(r'^advising/sis_semesters/(?P<jhed>.+)/$',
+        advising.views.StudentSISView.as_view()),
+
+    # Get the courses for the requesting user
+    url(r'^advising/sis_courses/(?P<sem_name>.+)/(?P<year>[0-9]{4})/$',
+        advising.views.RegisteredCoursesView.as_view()),
+    # Get the courses for the student with this JHED
+    url(r'^advising/sis_courses/(?P<sem_name>.+)/(?P<year>[0-9]{4})/(?P<jhed>.+)/$',
+        advising.views.RegisteredCoursesView.as_view()),
+    # Get the courses for the student with this JHED and verify it against this student's timetable
```

Comment:
there were some small issues with the URLs in practice, but i fixed them in my PR :)